extensions/objects/templates/math-editor.directive.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for position of terms editor.
+ */
+
+// Every editor directive should implement an alwaysEditable option. There
+// may be additional customization options for the editor that should be passed
+// in via initArgs.
+
+angular.module('oppia').directive('mathEditor', [
+  function() {
+    return {
+      restrict: 'E',
+      scope: {},
+      bindToController: {
+        value: '='
```

Comment:
Isn't 'value' being used here?

Reply:
Oh, yes, I missed this part. I've bound the value from the editor to this variable.